test/controllers/api/v2/content_views_controller_test.rb

```diff
@@ -85,6 +86,14 @@ def test_create
       assert_template 'katello/api/v2/common/create'
     end
 
+    def test_create_with_version_params
+      post :create, params: { :name => "My View", :label => "My_View", :major => 2, :minor => 1, :description => "Cool", :organization_id => @organization.id }
```

Comment:
this should be publish not create.

Reply:
I think the right approach for a controller test which goes to one or more backend services is simply to test that the action is invoked properly. Testing that the major & minor versions exist is the responsibility of a different test (in my opinion!).